.github/workflows/ci.yml

```diff
@@ -103,8 +103,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
-        nautobot-version: ["1.0.1"]
+        python-version: ["3.7", "3.8", "3.9"]
+        nautobot-version: ["1.2.11","1.3.3"]
```

Comment:
Any particular reason for 1.3.3 rather than 1.3.7?

Reply:
At the start of this issue, only 1.3.3 was released 😂